src/components/Progress/Progress.scss

```diff
@@ -83,6 +84,10 @@ $progress-thinnest-height: 4px;
         &_theme_misc {
             background-color: var(--yc-color-infographics-misc-medium);
         }
+
+        &_loading {
+            @include mixins.loading();
```

Comment:
Let's add `text-shadow: 0 0 1px var(--yc-color-text-dark-primary)` to the item's text. It will add more contrast.

Reply:
Maybe the text-shadow is only needed for certain colours? I think the contrast has rather decreased:

<img width="600" alt="shadow" src="https://github.com/gravity-ui/uikit/assets/529247/2fb67a75-6f33-4e12-872d-37d9d0929d21">

This is how this bar looks without the shadow, much better I think:

<img width="600" alt="no-shadow" src="https://github.com/gravity-ui/uikit/assets/529247/2e852d18-0064-4ac5-a4fe-5a2a13cfceb1">